### ISSUE TITLE
gitian-verify: torrent check

### ISSUE THREAD

**Issue by Sjors** [bitcoin-core/bitcoin-maintainer-tools#98]
It's a bit tedious now to check the contents of the torrent file.

**Comment by laanwj** [comment]
Yes, I think #74, being able to use it to verify downloaded binaries, would be the first step.
